projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -116,6 +116,10 @@ public function admin_init() {
 	 * Enqueue plugin admin scripts and styles.
 	 */
 	public function enqueue_admin_scripts() {
+		$screen = get_current_screen();
+		if ( ! empty( $screen ) && $screen->is_block_editor() ) {
+			return;
```

Comment:
What's the significance of this return and why do we need it?

Reply:
It prevents the admin page JS from loading in the block editor, so it doesn't load unnecessarily. @pablinos do you think we should take it a step further and only load these scripts on the actual Jetpack Social admin page?

Reply:
We could do. I was thinking it might be needed for other admin pages like Jetpack, or the classic editor. I wanted to prevent it running for the block editor mainly, as we have a very different setup there.

Reply:
Having said that, this is just for the admin page I think so we could go that way!

Reply:
Addressed in fe1944069e